Index: src/entity/HoaDon.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package entity;\n\npublic class HoaDon {\n\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/entity/HoaDon.java b/src/entity/HoaDon.java
--- a/src/entity/HoaDon.java	(revision 0f7cc61dabf7b670ad8bffcfac4bb7c95a8f2d2e)
+++ b/src/entity/HoaDon.java	(date 1761486922862)
@@ -1,6 +1,11 @@
 package entity;
 
 public class HoaDon {
-
-
+    private String maHoaDon;
+    private String hoTenKH;
+    private String soCCCD;
+    private String soDT;
+    private String nhanVienLapHD;
+    private double khuyenMai;
+    private double tongTien;
 }
Index: src/dao/HoaDonDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package dao;\n\npublic class HoaDonDAO {\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/dao/HoaDonDAO.java b/src/dao/HoaDonDAO.java
--- a/src/dao/HoaDonDAO.java	(revision 0f7cc61dabf7b670ad8bffcfac4bb7c95a8f2d2e)
+++ b/src/dao/HoaDonDAO.java	(date 1761465776286)
@@ -1,4 +1,5 @@
 package dao;
 
 public class HoaDonDAO {
+
 }
